docker/common.inc.sh

```diff
@@ -312,3 +312,36 @@ deploy() {
 run() {
   exec_scripts "$WORKDIR/docker/entry.d/" "container bootstrap"
 }
+
+show_tests_help() {
+  echo "
+SYNOPSIS: Initializes the test environment in an easier way.
+
+USAGE: $0 $1 [<suite>[,<suite>,<suite>,...]]
```

Comment:
please define `$0` and `$1` with a more meaningful name and `local` modifier like:

```
local RUN_CMD="$1"
```